src/aggregator/src/main/java/au/org/emii/util/ParameterRange.java

```diff
@@ -0,0 +1,14 @@
+package au.org.emii.util;
+
+
+public class ParameterRange {
+
+    public static String start;
+    public static String end;
+
+    public ParameterRange(String start, String end) {
+        this.start = start;
```

Comment:
Yep - like this, but why not use a double?

Reply:
So looking at the code this was being used internally only.  The aggregation worker was converting strings to data ranges, bounding boxes etc - thats probably where the vertical subset strings should be converted to a numeric range to be passed to the NetCDF Aggregator as per the other subset parameters.